src/Compilers/CSharp/Test/Syntax/Parsing/LambdaAttributeParsingTests.cs

```diff
@@ -1467,40 +1467,80 @@ public void As()
         [Fact]
         public void ConditionalExpression_01()
         {
-            UsingExpression("x ? [A] () => { } : z", TestOptions.RegularPreview,
-                // (1,1): error CS1073: Unexpected token '=>'
-                // x ? [A] () => { } : z
-                Diagnostic(ErrorCode.ERR_UnexpectedToken, "x ? [A] ()").WithArguments("=>").WithLocation(1, 1));
```

Comment:
note: this error appears to be incorrect (and the new test `ConditionalExpression_01_A` shows why that is).  It looks like we missed this when we added attribute-lambdas (or just accepted that it didn't work).  With the updated logic, this now properly parses.